codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordEncoder.java

```diff
@@ -101,4 +163,30 @@ protected void encodeName(String name, ByteBuf buf) throws Exception {
 
         buf.writeByte(0); // marks end of name field
     }
+
+    // Package private so it can be reused in the test.
+    static byte padWithZeros(byte b, int lowOrderBitsToPreserve) {
+        switch (lowOrderBitsToPreserve) {
+            case 0:
```

Comment:
Didn't we use this style before:

    switch (...) {
    case ...: // No extra indent
    }